Automattic/vip-workflow-plugin
# Add Tracks integration

## Description

Add Tracks integration into the plugin. This is dependent on the following MU-plugins PR: https://github.com/Automattic/vip-go-mu-plugins/pull/5770

## Steps to Test

1. Check out the MU-plugin PR above.
2. Check out this PR into the `plugins` directory of a local [VIP Go Skeleton repo](https://github.com/Automattic/vip-go-skeleton)
3. Create a new VIP dev-env with `vip dev-env --slug workflow-test --app-code={path to vip-go-skeleton repo} --mu-plugins={path to MU-plugins repo}`
4. Start the dev-env with `vip dev-env --slug workflow-test start`
5. Test the plugin
